boilerplates/trpc/files/$vite.config.ts.ts

```diff
@@ -6,6 +6,7 @@ export default async function getViteConfig(props: TransformerProps) {
   addVitePlugin(mod, {
     from: "./trpc/vite-plugin",
     constructor: "trpc",
+    index: props.meta.BATI.has("hono") ? 0 : undefined,
```

Comment:
@magne4000, 
When we use trpc vite-plugin with hono, we can't put it as the last plugin or we will get an error.

What do you think, it this ok?

Reply:
Is this behavior documented somewhere? Adding a comment just above explaining why it's necessary could be a good idea.
But if it works, we keep it yes.

Reply:
Thanks.
I don't know why, but it actually works, I'll revert this changes.